setup.py

```diff
@@ -19,7 +19,8 @@
     vertica.vertica_python = sqla_vertica_python.vertica_python:VerticaDialect
     """,
     install_requires=[
-        'vertica_python'
+        'vertica_python',
+        'psycopg2'
```

Comment:
Which addition requires `psycopg2`? I didn't see any direct references

Reply:
You are right. This should go in vertica_python instead.